esp/scm/ws_topology.ecm

```diff
@@ -532,6 +532,7 @@ ESPrequest [nil_remove] TpLogFileRequest
     [min_ver("1.05")] bool Zip(false);
     [min_ver("1.05")] int PageNumber(0);
     [min_ver("1.05")] bool LoadData(false);
+    [min_ver("1.29")] bool IncludeLogFieldNameLine(true);
```

Comment:
Does the caller need to know it is grabbed from a line?  That is would just "IncludeLogFieldNames" be a better name?